### ISSUE TITLE
[rllib] QMIX: No backprop on the (mixer) hypernetworks?

### ISSUE THREAD

**Issue by rohitboakde94** [ray-project/ray#5003]
### Source code / logs
        # Setup optimizer
        self.params = list(self.model.parameters())
        self.loss = QMixLoss(self.model, self.target_model, self.mixer,
                             self.target_mixer, self.n_agents, self.n_actions,
                             self.config["double_q"], self.config["gamma"])

**Comment by ericl** [comment]
It looks like this is missing from the port: `self.params += list(self.mixer.parameters())`
